freqtrade/data/converter.py

```diff
@@ -37,9 +40,29 @@ def parse_ticker_dataframe(ticker: list, timeframe: str, pair: str, *,
     # and fail with exception...
     frame = frame.astype(dtype={'open': 'float', 'high': 'float', 'low': 'float', 'close': 'float',
                                 'volume': 'float'})
+    return clean_ohlcv_dataframe(frame, timeframe, pair,
+                                 fill_missing=fill_missing,
+                                 drop_incomplete=drop_incomplete)
 
+
+def clean_ohlcv_dataframe(data: DataFrame, timeframe: str, pair: str, *,
```

Comment:
`*, `
-- is not this a syntax sugar added in python 3.8 only?
(https://docs.python.org/3/whatsnew/3.8.html#positional-only-parameters)

Reply:
no - that is `/` - to force positional only parameters (these cannot be specified using kwargs - allowing the function to rename parameters at future points).
Now i don't really see a point in using positional only arguments - but i guess it can make sense for certain usecases.

Either way - this is the reverse - meaning every argument after the * must be specified using the keyword.

This syntax (kwargs only after *) has been introduced in [PEP-3102](https://www.python.org/dev/peps/pep-3102/) - and seems to have been there since python3.0 (although I've not used it until python 3.6).

Anyway, we support 3.6 and up (and also run Ci on that, so this is valid syntax ( i even used it before in `converter.py` - so this is not entirely new to freqtrade).

Reply:
Ah, the opposite... Do not use it, so just do not remember it in python 3.6+,
thanks!